src/components/timeline/ActiveTabGlobalTrack.js

```diff
@@ -6,23 +6,23 @@
 
 import React, { PureComponent } from 'react';
 import classNames from 'classnames';
-import { selectActiveTabTrack } from '../../actions/profile-view';
+import { selectActiveTabTrack } from 'firefox-profiler/actions/profile-view';
 import {
   getSelectedThreadIndexes,
   getSelectedTab,
-} from '../../selectors/url-state';
-import explicitConnect from '../../utils/connect';
+} from 'firefox-profiler/selectors/url-state';
+import explicitConnect from 'firefox-profiler/utils/connect';
 import {
   getActiveTabGlobalTracks,
   getActiveTabResourceTracks,
-} from '../../selectors/profile';
-import './Track.css';
-import TimelineTrackThread from './TrackThread';
-import TimelineTrackScreenshots from './TrackScreenshots';
-import ActiveTabResourcesPanel from './ActiveTabResourcesPanel';
```

Comment:
Sorry that it wasn't clear: in the case of the components in the same directory (or in a subdirectory in the current directory), it's better to keep `./`. What we want to change is when we use `../`, to move __up__ in the directory hierarchy.